NewForce-Cohort-6/tabloidfullstack-mashed-potatoes
# 30SubscribeToAUsersPosts

# Description
[Trello Ticket #30](https://trello.com/c/9cztYRV1)

Added functionality to subscribe to another user's posts. When viewing a post that a user has not subscribed to, a Subscribe button will appear. When viewing a post by a user that you have subscribed to, it will be noted that you are already subscribed. When you are viewing a post by the logged-in user, neither will be visible.


# Testing Instructions
1. `git add` and `git commit` whatever you're working on on your branch
2. `git checkout main`
3. `git fetch --a`
4. `git checkout 30SubscribToAUsersPosts`
5. If you don't see the most recent version of my files, `git pull origin 830SubscribToAUsersPostsEditAPost`
6. If you haven't created the Tabloid database, please do so.
Use this data to fill out tables and columns: [SQL Tables](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/01_Tabloid_Create_DB.sql)
Use this to seed the database : [SQL SEED DATA](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/02_Tabloid_Seed_Data.sql)
7. Click the execute with debugger solid green button to launch the app
8. If you are not already logged in to the site, log in.
9. From the home page, click the “My Posts” or "Posts" option in the navbar and select a post by another user.
10. You should be redirected to that post's details page and there should be a subscribe button at the top
11. Click subscribe
12. The button should be replaced with a note indicating you're subscribed 
13. Go back to the post list and select another post by a user you've subscribed to. You'll be directed to it's post details page and there will be a note indicating you're subscribed 
14. Go back to the post list and select a post by the user you've logged in with. There should not be a button or a note about subscription status on this page

# Checklist:
